settlement/local/local.go

```diff
@@ -54,6 +54,16 @@ type Client struct {
 	settlementKV store.KV
 }
 
+func (c *Client) GetStateInfo(index uint64) (*types.StateInfo, error) {
```

Comment:
not ncessary since we only use to get next proposer, and there is already a function for it